Jantimizer/QueryOptimiserTest/QueryGeneratorTests/PostgresGeneratorTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Histograms.Managers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using QueryOptimiser.QueryGenerators;
+using QueryOptimiserTest;
+using QueryParser;
+using QueryParser.QueryParsers;
+using QueryParser.Models;
+using System;
+using System.Collections.Generic;
+
+namespace GeneratorTest;
+
+[TestClass]
+public class GeneratorTest
```

Comment:
This isnt tests for the Generator folders now is it.
Should be renamed to `PostgresGeneratorTests` instead maybe?